src/molecules/input/Input.stories.js

```diff
@@ -0,0 +1,130 @@
+import AInput from './Input.vue'
+import AIcon from '../../atoms/icon/Icon.vue'
+import ALabel from '../../atoms/label/Label.vue'
+import AIconPerson from '../../atoms/icon/templates/IconPerson.vue'
+
+export default {
+  title: 'Molecules/Input',
+  component: AInput,
+  argTypes: {
+    id: {
+      control: {
+        type: 'text'
+      }
+    },
+    label: {
+      control: {
+        type: 'text'
+      }
+    },
+    variant: {
+      control: {
+        type: 'select',
+        options: [
+          'primary',
+          'inline'
+        ]
+      }
+    }
+  }
+}
+const defaultArgs = {
+  id: 'input-text',
+  label: 'input label',
+  variant: 'primary'
+}
+
+const Template = (args, { argTypes }) => ({
+  components: { AInput },
+  props: Object.keys(argTypes),
+  data: () => {
+    return {
+      inputValue: ''
+    }
+  },
+  template: `
+    <div>
+      <a-input
+        v-model="inputValue"
+        :id="id"
+        :label="label"
+        :variant="variant"
+      />
+    </div>
+  `
+})
+
+export const Default = Template.bind({})
+Default.args = defaultArgs
+
+export const WithSlotsIcon = (args, { argTypes }) => ({
+  components: { AInput, AIcon, AIconPerson },
+  props: Object.keys(argTypes),
+  data: () => {
+    return {
+      inputValue: ''
+    }
+  },
+  template: `
+    <div>
+      <a-input
+        v-model="inputValue"
+        type="text"
+        :label="label"
+        :id="id"
+        :variant="variant"
+      >
+        <template #icon="{ variant }">
+          <div
+            :class="[
+              'h-12 w-12',
+              'flex items-center justify-center',
+              'absolute right-0',
+              variant === 'primary' ? 'top-6' : 'top-0'
```

Comment:
I think it might be useful to have a variant in config for this. I can see also in designs that we have a seach input with icon on the left. Another slot and variant might be needed. 
![image](https://user-images.githubusercontent.com/49198312/125950483-7d55791c-a5df-42f5-9513-2520489fa487.png)